README.md

```diff
@@ -507,6 +509,526 @@ source scripts/makeAllPlots.sh figures/ResponseUniformityMaps
+
+What you have to do is:   
+
+```
+python2.7 python/Produce_Config_File.py —-file=QC3_GE11-X-L-CERN-0002_2017_06_01.xlsm —-SelectSheetNum=0 —-SelectColumnX=1 —-SelectColumnY=2 --SelectRowStart=1 --SelectRowEnd=61   
+```
+The options you have set are:
+						
+* `—-file=QC3_GE11-X-L-CERN-0002_2017_06_01.xlsm`	
+
+(the desired file stored in the base dir of the repository.
+If you want to plot the comparison of multiple files you can write:  
+
+`—-file=NameofFirstFile.xlsm —-file=NameofSecondFile.xlsm --file=NameofThirdFile.xlsm`),
+	
+	
+* `—-SelectSheetNum=0`
+
+	(the excel sheet number starting counting from zero),
+
+* `—-SelectColumnX=1 —-SelectColumnY=2`
+	
+	(The desired columns for read starting counting from zero),
+
+* `--SelectRowStart=1 --SelectRowEnd=61`
+
+	(The desired rows for read starting counting from zero).
+	
+After executing the command above, a **.cfg** file will be created in the base directory of the repository. If the user doesn't select the name of the **.cfg** file by adding the option **--OutputName=SomeName** in the command line above, the default name **Myconfig.cfg** will be set. In addition all the default values described in the options table above will be set.
+
+The produced config file in our example is shown below:
+
+**Myconfig.cfg:**   
+
+	[BEGIN_CANVAS]
+	Canv_Axis_NDiv = '508,510';#X,Y
+	Canv_Dim= '1000,1000';#X,Y
+	Canv_DrawOpt = 'APE1';
+	Canv_Grid_XY = 'false,false';
+	Canv_Legend_Dim_X = '0.20,0.60';
+	Canv_Legend_Dim_Y = '0.56,0.92';
+	Canv_Legend_Draw = 'true';
+	Canv_Log_XY = 'false,false';
+	Canv_Logo_Pos = '0';
+	Canv_Logo_Prelim = 'true';
+	Canv_Margin_Top = '0.08';
+	Canv_Margin_Bot = '0.14';
+	Canv_Margin_Lf = '0.16';
+	Canv_Margin_Rt = '0.06';
+	Canv_Name = 'LS2_Detectors';
+	Canv_Plot_Type = 'TGraphErrors';
+	Canv_Range_X = '0,1000';
+	Canv_Range_Y = '0,7';
+	Canv_Title_Offset_X = '1.0';
+	Canv_Title_Offset_Y = '1.0';
+	Canv_Title_X = '';
+	Canv_Title_Y = '';
+		[BEGIN_PLOT]
+			Plot_Color = 'kRed';
+			Plot_LegEntry = 'GE11-X-L-CERN-0002';
+			Plot_Line_Size = '1';
+			Plot_Line_Style = '1';
+			Plot_Marker_Size = '0.8';
+			Plot_Marker_Style = '20';
+			Plot_Name = 'GE11-X-L-CERN-0002';
+			[BEGIN_DATA]
+			VAR_INDEP,VAR_DEP,VAR_INDEP_ERR,VAR_DEP_ERR
+			1.0,20.38,0.0,0.0
+			62.0,20.26,0.0,0.0
+			123.0,20.13,0.0,0.0
+			184.0,20.14,0.0,0.0
+			245.0,20.11,0.0,0.0
+			306.0,20.11,0.0,0.0
+			367.0,20.04,0.0,0.0
+			428.0,20.05,0.0,0.0
+			489.0,20.03,0.0,0.0
+			550.0,20.07,0.0,0.0
+			...................
+			...................
+			...................
+			2990.0,19.39,0.0,0.0
+			3051.0,19.33,0.0,0.0
+			3112.0,19.39,0.0,0.0
+			3173.0,19.32,0.0,0.0
+			3234.0,19.32,0.0,0.0
+			3295.0,19.35,0.0,0.0
+			3356.0,19.23,0.0,0.0
+			3417.0,19.25,0.0,0.0
+			3478.0,19.25,0.0,0.0
+			3539.0,19.29,0.0,0.0
+			3600.0,19.2,0.0,0.0
+			[END_DATA]
+		[END_PLOT]
+	[END_CANVAS]
```

Comment:
Lines 612-675, remove.  Examples of plot configs are already given in Section 4.e.iv, for example reference the user to [here](https://github.com/bdorney/CMS_GEM_Analysis_Framework/#4eivvi-example-config-file---tgraph)

Reply:
done..